apps/builder/app/builder/features/settings-panel/variable-popover.tsx

```diff
@@ -427,7 +437,20 @@ const VariablePanel = forwardRef<
           "A Resource is a configuration for secure data fetching. You can safely use secrets in any field.",
       },
     ],
+    [
+      "system-resource",
+      {
+        label: (
+          <Flex direction="row" gap="2" align="center">
+            System Resource
+            {allowDynamicData === false && <ProBadge>Pro</ProBadge>}
+          </Flex>
+        ),
+        description: "A System Resource is a configuration for system data.",
```

Comment:
not sure if much better, but seems like more clear which system it is

```suggestion
        description: "A System Resource is a configuration for Webstudio data.",
```